src/main/java/uk/gov/laa/ccms/caab/controller/HomeController.java

```diff
@@ -1,7 +1,9 @@
 package uk.gov.laa.ccms.caab.controller;
 
+import static uk.gov.laa.ccms.caab.constants.SessionConstants.APPLICATION_ID;
```

Comment:
Is this used in this class?

Reply:
It is not i used it when testing, will remove